client/gutenberg/extensions/mailchimp/edit.jsx

```diff
@@ -183,43 +184,29 @@ class MailchimpSubscribeEdit extends Component {
 		);
 		const blockContent = (
 			<div className={ className }>
-				<RichText
-					tagName="h3"
-					placeholder={ __( 'Write title' ) }
-					value={ title }
-					onChange={ value => setAttributes( { title: value } ) }
-					inlineToolbar
-				/>
 				{ ! audition && (
 					<form ref={ this.formRef }>
-						<TextControl placeholder={ emailPlaceholder } onChange={ () => false } />
-						<Button isPrimary>{ submitLabel }</Button>
+						<TextControl placeholder={ emailPlaceholder } onChange={ () => false } type="email" />
+						<div className="wp-block-button__link wp-block-jetpack-mailchimp_button">
```

Comment:
Was it impossible to use the `Button` component here? I'd very much prefer to rely on that component rather than a class it uses.

If we need the styling, I think we're better off pulling the necessary styles into our own CSS.

Reply:
The goal was to emulate the Gutenberg button block in appearance and functionality - and since https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/button/edit.js doesn't make use of the `Button` component I fear that this would lead to slight style divergences and then CSS to overcome them. Re: the styling, yes I'll pull it in.

Reply:
> If we need the styling, I think we're better off pulling the necessary styles into our own CSS.

This is done in https://github.com/Automattic/wp-calypso/pull/30685/commits/8f066fdf9b2e425589ab12a006d474d7c1376bda